ingestion/src/metadata/ingestion/source/database/athena.py

```diff
@@ -84,6 +87,14 @@ def _get_column_type(self, type_):
         col_type = types.VARCHAR
         if length:
             args = [int(length)]
+    elif type_.startswith("array"):
+        parsed_type = (
+            ColumnTypeParser._parse_datatype_string(  # pylint: disable=protected-access
+                type_
+            )
+        )
+        col_type = col_map["array"]
+        args = [col_map.get(parsed_type.get("arrayDataType", "string").lower())]
```

Comment:
I think we are not handling this 100% correctly, ["arrayDataType"](https://github.com/open-metadata/OpenMetadata/blob/68c3e0b8feb98ce2f7af7f632e667e4ba98eda6a/ingestion/src/metadata/ingestion/source/database/column_type_parser.py#L232) will always be available if type starts with `array`, 


so instead of doing this
```
args = [col_map.get(parsed_type.get("arrayDataType", "string").lower())]
```

we should handle it something like this 
```
args = [col_map.get(parsed_type.get("arrayDataType").lower(),types.String)]
```

which basically means if the parsed type is not available in column map then pick up string as type

Reply:
Done.